jest.config.ts

```diff
@@ -37,14 +37,24 @@ const config: Config.InitialOptions = {
   moduleNameMapper: {
     '^jotai-devtools$': '<rootDir>/src/index.ts',
     '^jotai-devtools/(.*)$': '<rootDir>/src/$1.ts',
+    '\\.(css|less|sass|scss)$': '<rootDir>/__mocks__/styleMock.js',
   },
   modulePathIgnorePatterns: ['dist'],
   globals: {
     __DEV__: true,
```

Comment:
Just to let you know that Jotai repo goes with a new approach. https://github.com/pmndrs/jotai/pull/1742

Reply:
Nice! I'll revisit this after a few stable releases of `jotai-devtools`